### PR TITLE
Support JavaScript for writing source files

### PR DESCRIPTION
**Pull request by lipis**
So simply instead of creating `.coffee` files inside the `app/script` directory we can now create `.js` files and the rest remain the same.

### CODE REVIEW

File: grunt/config/copy.coffee [hunk 1]
```diff
@@ -34,9 +34,16 @@ module.exports =
...
       'style/*.css'
+      'script/*/*.js'
       'worker/*'
     ]
 
+  dist_js:
+    cwd: '<%= dir.app_ %>'
+    dest: '<%= dir.dist %>'
+    expand: true
+    src: 'script/*/*.js'
```
Comment (ffflorian) [inline grunt/config/copy.coffee]: Shouldn't it be `script/**/*.js`?

Comment (bennycode) [inline grunt/config/copy.coffee]: Shouldn't `script/*/*.js` become `script/**/*.js`?

File: grunt/config/copy.coffee [hunk 2]
```diff
@@ -64,6 +71,7 @@ module.exports =
       'image/**/*'
       'font/**/*'
       'style/*.css'
+      'script/*/*.js'
```
Comment (ffflorian) [inline grunt/config/copy.coffee]: Same here?

Comment (ffflorian) [inline grunt/config/copy.coffee]: Shouldn't it be `script/**/*.js`?

**Comment by herrmannplatz** [comment]
browsers!

**Pull request merged**